ppr-ui/src/views/mhrInformation/MhrTransportPermit.vue

```diff
@@ -76,7 +77,7 @@
 
           <!-- Default Transport Permit Actions -->
           <v-btn
-            v-else-if="!isExemptMhr"
+            v-else-if="!isExemptMhr && (!isAmendChangeLocationEnabled && !hasActiveTransportPermit)"
```

Comment:
Will this turn the Transport permit option off as well when the feature flag is `true`?

Reply:
This will turn off the ability to 'Amend Transport Permit' with a `false` flag.  The normal Transport Permit will not be affected.